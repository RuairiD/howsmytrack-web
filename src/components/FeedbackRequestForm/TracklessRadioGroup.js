import React from "react";

import { Radio } from "antd";

const TracklessRadioGroup = ({ decorator }) => decorator(
    <Radio.Group className="trackless-radio" buttonStyle="solid">
        <Radio.Button value="track">
            I have a track I would like feedback on
        </Radio.Button>
        <Radio.Button value="trackless">
            I don't have a track and would just like to provide feedback for others
        </Radio.Button>
    </Radio.Group>,
);

export default TracklessRadioGroup;
