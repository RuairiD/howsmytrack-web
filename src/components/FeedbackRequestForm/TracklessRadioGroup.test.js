import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import TracklessRadioGroup from "./TracklessRadioGroup";

describe("TracklessRadioGroup", () => {
    it("renders a RadioGroup for selecting whether the request is trackless or not", () => {
        const wrapper = shallow(
            <TracklessRadioGroup
                decorator={(x) => x}
            />,
        );
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
