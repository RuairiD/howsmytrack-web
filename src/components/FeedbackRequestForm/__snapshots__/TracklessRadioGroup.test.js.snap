// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TracklessRadioGroup renders a RadioGroup for selecting whether the request is trackless or not 1`] = `
<RadioGroup
  buttonStyle="solid"
  className="trackless-radio"
>
  <RadioButton
    value="track"
  >
    I have a track I would like feedback on
  </RadioButton>
  <RadioButton
    value="trackless"
  >
    I don't have a track and would just like to provide feedback for others
  </RadioButton>
</RadioGroup>
`;
